src/AddTodoForm.jsx

```diff
@@ -13,19 +14,17 @@ export default function AddTodoForm({ onAddTodo }) {
     e.preventDefault();
     onAddTodo({
       title: todoTitle,
-      id: Date.now()
+      id: Date.now(),
     });
-    document.getElementById('titleForm').reset();
+    document.getElementById("titleForm").reset();
   }
 
-
   return (
     <form id="titleForm" onSubmit={handleAddTodo} onChange={handleTitleChange}>
```

Comment:
I'd recommend passing this handleTitleChange function as a property of InputWithLabel, rather than the form element.